atlp-rwanda/Team-Sostene-E-commerce-fe
# #185172085 Configure storybook for component development

#### What does this PR do?
* creates storybook for using on related components
#### Description of Task to be completed?
* This PR allows the configuration of all features to use on storybook to allow the easy reusable of components
#### How should this be manually tested?
* After cloning the Repo user should create new storybook features and run them using `yarn storybook`
#### What are the relevant pivotal tracker/Trello stories?
#185172085
